.circleci/config.yml

```diff
@@ -55,6 +55,9 @@ workflows:
       - tests-java-test-app:
           requires:
             - java-app-encryption
+      - server-java:
+          requires:
+            - java-app-encryption
```

Comment:
since java app encryption is released, this should be un-necessary.
`server-java` should use the latest released version